chemicaltoolbox/autodock_vina/prepare_receptor/prepare_receptor.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="prepare_receptor" name="Prepare receptor" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.5.6">mgltools</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <description> - Tool for prepare receptor with Autodock Vina</description>
+    <command interpreter="python"><![CDATA[
+        prepare_receptor.py -r "$receptor" 
+                            -p "$file_output.files_path" 
+                            -f "$file_output"
+    ]]></command>
+    <inputs>
+        <param type="data" name="receptor" format="pdb" label="Receptor: " help="No data? See tip below" />
+    </inputs>
+    <outputs>
+        <data name="file_output" format="pdbqt" label="#echo os.path.splitext (str($receptor.name))[0]#.pdbqt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="receptor" value="3u1i_for_DM.pdb"/>
+            <output name="file_output" file="3u1i_for_DM.pdbqt"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+        *** Tool for prepare receptor with Autodock Vina ***
```

Comment:
This could be extended a little bit ... `Tool to prepare a receptor (Protein) ...` maybe?